FrontEnd/scripts/works.js

```diff
@@ -50,31 +56,73 @@ async function displayWork(worksCategory) {
     workImage.setAttribute("alt", work.title);
     workFigCaption.textContent = work.title;
 
-    // NOTE: adds to elements to the parent html container "gallery" cold before
+    // NOTE: adds to elements to the parent html container called before
     workFigure.appendChild(workImage);
     workFigure.appendChild(workFigCaption);
-    worksContainer.appendChild(workFigure);
+    target.appendChild(workFigure);
   });
 }
 
+// NOTE: Ici commence l'execution du code
+//
+//
+if (sessionStorage.getItem("token") != null) {
+  editionModeBand.style.display = "flex";
+  for (editionButton of editionModeButtons) {
+    editionButton.style.display = "flex";
+  }
+}
+
+// NOTE: opens the modal
+modifyBtn.addEventListener("click", () => {
+  modalBox.style.display = "flex";
+  modalBox
+    .querySelector(".modalCloseBtn")
+    .addEventListener("click", closeModal);
+  modalBox
+    .querySelector(".modalStop")
+    .addEventListener("click", stopPropagation);
+});
+
+// NOTE: CLoses the modal
+const closeModal = function (event) {
+  if (modalBox === null) return;
```

Comment:
We should have the same in the open modal one, WDYT?

Reply:
i dont really see why we should have something like this because to do that we need to click outside the modal so the modal will close anyway before opening it again

Reply:
Maybe: 

```js
if (modalBox === null || modalBox.style.display = "flex") return;
```

I mean technically it could still happen as the function could be used somewhere else; you just prefer to save your ass. It's more a good practice in general.